.pre-commit-config.yaml

```diff
@@ -22,8 +22,20 @@ repos:
       additional_dependencies:
       - ethlint@1.2.3
     - id: lint-js
-      name: 'lint js'
+      name: 'lint solidity js'
       entry: /usr/bin/env bash -c "cd solidity && npm run lint:js"
-      files: '\.js$'
+      files: 'solidity\/.*\/\.js$'
+      language: script
+      description: "Checks JS code according to the package's linter configuration"
+    - id: lint-dashboard-js
+      name: 'lint dashboard js'
+      entry: /usr/bin/env bash -c "cd solidity/dashboard && npm run js:lint"
+      files: 'solidity\/.*\/\.js$'
```

Comment:
Don't we need to update this filter to respect the dashboard's file structure?

Reply:
… I swear I'd set this one up right 😅

Reply:
Should be fixed :grimacing: